tests/infer/test_conjugate_gradients.py

```diff
@@ -0,0 +1,31 @@
+
+import pyro
+from pyro.infer.tracegraph_elbo import TraceGraph_ELBO
+from tests.common import assert_equal
+from tests.integration_tests.test_conjugate_gaussian_models import GaussianChain
+
+
+# TODO increase precision and number of particles once latter is parallelized properly
+class ConjugateChainGradientTests(GaussianChain):
+
+    def test_gradients(self):
```

Comment:
Unfortunately, `pytest.parametrize` doesn't mix well with `unittest.TestCase`. There are some work-arounds mentioned in https://docs.pytest.org/en/latest/unittest.html#mixing-fixtures, but your solution here is preferable. We should just consider migrating all `unittest.TestCase` tests to pytest at some point.

Reply:
It doesn't work because `GaussianChain` inherits off of `unittest.TestCase`, and that can only run tests that do not have any arguments (pytest internally just calls `TestCase.runTest()`). We can change `GaussianChain` to inherit off of `object`, and then this would run, but we will need to manually call the `setUp` method here and in the other tests in `test_conjugate_gaussian_models`.